src/components/Slider/Slider.vue

```diff
@@ -196,9 +242,32 @@ export default {
 <style lang="scss">
 @import '../../assets/scss/unnnic.scss';
 
+.unnnic-label {
+      display: flex;
+      gap: $unnnic-spacing-xs;
+
+    &__tooltip {
+        display: flex !important;
```

Comment:
Why are these `!important` needed here? Using `!importart` is bad practice.
Look for some way around this or if it is really necessary, please add a comment in the code justifying the reason.

Reply:
I just kept what is being used. It may be that in some situation it must have broken. Anyway, I made the adjustments.